str/helper/compute_gc_metrics.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+"""
+Calculates GC metrics for a CRAM file using Picard CollectGcBiasMetrics.
+
+analysis-runner --dataset "bioheart" \
+    --description "Compute GC metrics" \
+    --access-level "test" \
+    --output-dir "qc-stand-alone" \
+    compute_gc_metrics.py --input-dir "gs://cpg-bioheart-test/cram" CPGXXXX
+
+
+"""
+import os
+import click
+
+from cpg_utils.hail_batch import command, image_path, get_batch, output_path
+from cpg_workflows.resources import HIGHMEM, STANDARD
+
+
+REF_FASTA = 'gs://cpg-common-main/references/hg38/v0/dragen_reference/Homo_sapiens_assembly38_masked.fasta'
+
+
+# input directory to CRAMs
+@click.option('--input-dir', help='gs://...')
+# input CPG sample ID
+@click.argument('internal-wgs-ids', nargs=-1)
+@click.command()
+def main(input_dir, internal_wgs_ids):
+    """
+    Make job that runs Picard CollectGcBiasMetrics.
+    """
+    b = get_batch(name='Compute GC metrics')
+    reference = b.read_input_group(
+        base=REF_FASTA,
+        fai=REF_FASTA + '.fai',
+        dict=REF_FASTA.replace('.fasta', '').replace('.fna', '').replace('.fa', '')
+        + '.dict',
+    )
+
+    cram_dict = {id: os.path.join(input_dir, f'{id}.cram') for id in internal_wgs_ids}
+
+    for id in list(cram_dict.keys()):
+
+        input_cram = b.read_input(cram_dict[id])
+
+        j = b.new_job(f'CollectGcBiasMetrics: {id}')
+
+        j.image(image_path('picard'))
+        resource = STANDARD.request_resources(ncpu=1)
+        resource.attach_disk_storage_gb = 50
+        resource.set_to_job(j)
```

Comment:
You can make this storage reservation super tight:

```
from cpg_utils import to_path

cram_file = to_path(cram_dict[id])
cram_size = cram_file.stat().st_size
job.storage(cram_size * 1.5)  # arbitrary amount of storage as a buffer

Reply:
good idea, thanks!